zipkin-server/src/main/java/zipkin2/server/internal/RegisterZipkinHealthIndicators.java

```diff
@@ -13,23 +13,32 @@
  */
 package zipkin2.server.internal;
 
+import java.util.LinkedHashMap;
+import java.util.Map;
 import org.springframework.beans.factory.config.ConfigurableListableBeanFactory;
+import org.springframework.boot.actuate.health.CompositeHealthIndicator;
+import org.springframework.boot.actuate.health.HealthAggregator;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.boot.actuate.health.HealthIndicatorRegistry;
 import org.springframework.boot.context.event.ApplicationReadyEvent;
 import org.springframework.context.ApplicationEvent;
 import org.springframework.context.ApplicationListener;
 import zipkin2.Component;
 
 /** Makes sure all zipkin components end up in the /health endpoint. */
 // This is an application listener to ensure the graph is fully constructed before doing health
-public final class RegisterZipkinHealthIndicators implements ApplicationListener {
```

Comment:
this was full of deprecated calls so I rejigged until nothing deprecated anymore